google/cloud/firestore_v1/base_aggregation.py

```diff
@@ -230,6 +234,8 @@ def get(
             retries.Retry, None, gapic_v1.method._MethodDefault
         ] = gapic_v1.method.DEFAULT,
         timeout: float | None = None,
+        *,
+        explain_options: Optional[ExplainOptions] = None,
     ) -> List[AggregationResult] | Coroutine[Any, Any, List[AggregationResult]]:
```

Comment:
update the return type? (looks like there are a lot that can be changed across the files)

Reply:
done, also checked the whole file